int16 left
int16 right
int16 camservo
string fw_version
